CONTRIBUTING.md

```diff
@@ -46,11 +46,62 @@ In general, we follow the ["fork-and-pull" Git workflow](https://github.com/susa
 
 1. Fork the repository to your own Github account
 2. Clone the project to your machine
-3. Create a branch locally with a succinct but descriptive name
-4. Commit changes to the branch
-5. Following any formatting and testing guidelines specific to this repo
-6. Push changes to your fork
-7. Open a PR in our repository and follow the PR template so that we can efficiently review the changes.
+```
+git clone https://github.com/divanov11/Mumble.git
+```
+3. Add Upstream or the remote of the original project to your local repository
+```
+# check remotes
+git remote -v
+git remote add upstream https://github.com/divanov11/Mumble.git
+```
+4. Make sure you update the local repository
+```
+# Get updates
+git fetch upstream
+# switch to master branch
+git checkout master
+# Merge updates to local repository
+git merge upstream/master
+# Push to github repository
+git push origin master
+```
+5. Create a branch locally with a succinct but descriptive name
+```
+git checkout -b branch-name
+```
+6. Commit changes to the branch
+```
+# Stage changes for commit i.e add all modified files to commit
+git add .
+git commit -m "added git commands for "fork-and-pull" Git workflow"
+# check status
+git status
+```
+7. Following any formatting and testing guidelines specific to this repository
+8. Push changes to your fork
+```
+git push origin branch-name
+```
+9. Open a PR in our repository and follow the PR template so that we can efficiently review the changes.
+10. After the pull request was merged, fetch it and update the default branch of your fork
```

Comment:
```suggestion
10. After the pull request was merged, fetch the upstream and update the default branch of your fork
```
Maybe this can be slightly rephrased?